src/generator/operations.ts

```diff
@@ -105,11 +105,15 @@ export async function generateOperations(session: Session<CodeModel>): Promise<O
 }
 
 // use this to generate the code that will help process values returned in response headers
-function formatHeaderResponseValue(propName: string, header: string, schema: Schema, imports: ImportManager, respObj: string, zeroResp: string): string {
+function formatHeaderResponseValue(propName: string, header: string, schema: Schema, imports: ImportManager, respObj: string, zeroResp: string, multPrefix: boolean): [string, boolean] {
   // dictionaries are handled slightly different so we do that first
   if (schema.type === SchemaType.Dictionary) {
     imports.add('strings');
-    let text = `\tprefix := strings.ToUpper("${schema.language.go!.headerCollectionPrefix}")\n`;
+    let firstPrefix = ':';
```

Comment:
```suggestion
   const prefix = multiPrefix? ":" : "";
```

or even changing to be more descriptive and updating below

```suggestion
   const operator = multiPrefix? ":=" : "=";
```

Reply:
Done! Thank you for the suggestion, I like it a lot more than what I had.